packages/gluestick/src/commands/start.js

```diff
@@ -16,6 +20,7 @@ type StartOptions = {
   logLevel?: string;
   runTests: boolean;
   skipBuild: boolean;
+  noDepCheck: boolean;
```

Comment:
This should be `depCheck` - Commander negates options starting with `no`:
- if `--no-dep-check` **is not** passed => `depCheck === true`
- if `--no-dep-check` **is** passed => `depCheck === false`